frontend/packages/operator-lifecycle-manager/src/components/operand/index.tsx

```diff
@@ -397,15 +398,13 @@ export const ProvidedAPIsPage = (props: ProvidedAPIsPageProps) => {
 
   const createItems =
     providedAPIs.length > 1
-      ? providedAPIs.reduce((acc, api) => ({ ...acc, [api.name]: api.displayName }), {})
+      ? providedAPIs.reduce(
+          (acc, api) => ({ ...acc, [referenceForProvidedAPI(api)]: api.displayName }),
+          {},
+        )
       : {};
 
-  const createNavigate = (name) =>
-    history.push(
-      `/k8s/ns/${obj.metadata.namespace}/${ClusterServiceVersionModel.plural}/${
-        obj.metadata.name
-      }/${referenceForProvidedAPI(_.find(providedAPIs, { name }))}/~new`,
-    );
+  const createNavigate = (kind) => history.push(`${props.match.url}/${kind}/~new`);
```

Comment:
Is `kind` here the full reference as before?

Reply:
Yes, the arg passed to this callback is a key from the `crateItems` object. I changed those keys to the full API reference on L402.